scripts/query_config

```diff
@@ -23,7 +23,7 @@ def query_grids(long_output):
     """
     query all grids.
     """
-    files = Files()
+    files = Files(comp_interface="nuopc")
```

Comment:
This seems wrong. Shouldn't query config take the comp_interface as an option?

Reply:
fixed